bootcamp/src/topic0/DuplexBuilder.java

```diff
@@ -0,0 +1,27 @@
+package topic0;
+
+public class DuplexBuilder implements BuildingBuilder {
+
+private Building building;
+
+		public DuplexBuilder() {
+			building = new Building();
+		}
+
+		@Override
+		public void buildAddress() {
+			building.setAddressNumber(42);;
+			building.setStreet("St Monica");
+		}
+		@Override
+		
+		public void buildStructure(){
+			building.setNFloors(2);
```

Comment:
At this code, we are asuming that ALWAYS adding that number or floors, rooms and the same address for all the buildings... need to pass this as a param, in order to be more flexible